manifests/kubeweekly-jobs.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kubeweekly-jobs
+spec:
+  schedule: "*/12 * * * 3"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: kubeweekly
+            image: kubeweekly:latest
+            args:
+            - npm run trigger
+          restartPolicy: OnFailure
```

Comment:
it's better to add newline in the end

Reply:
https://github.com/cloudnative-id/kubeops/pull/3/commits/f08dc065397332554b19424289a1ca70a06d156d